disnake/entitlement.py

```diff
@@ -76,7 +76,7 @@ class Entitlement(Hashable):
         Set to ``None`` when this is a test entitlement.
     ends_at: Optional[:class:`datetime.datetime`]
         The time at which the entitlement stops being active.
-        Set to ``None`` when this is a test entitlement.
+        Set to ``None`` when this is a test entitlement or when this is an indefinite entitlement.
```

Comment:
```suggestion
```
At this point, it feels like this sentence isn't needed at all anymore (I think?). The first part of the description already explains it, now that there's no special behavior for `ends_at` anymore.